ls1intum/artemis-android
# `Bugfix`: Fix image jittering in chat for all images

### Problem Description

After applying #109 there was a problem that caused the chat to jitter when reloading. This was caused by images smaller than the default height, as described in #166.

### Changes

The root of the problem was unnecessary reloads that were triggered when scrolling through the list. Therefore, a condition has been introduced checking if the markdown text has changed and only performing an update if there are changes. This PR fixes #166

### Steps for testing

1. Go to a chat that contains images (preferably images which have a greater width than height)
2. Perform scrolling and reloading the chat
3. Verify that there is no jittering
